INTITA/Modules/ForgotPassword/ForgotPasswordViewController.swift

```diff
@@ -74,8 +74,8 @@ class ForgotPasswordViewController: UIViewController, Storyboarded {
         let explanationCell = UINib(nibName: "ExplanationTableViewCell", bundle: nil)
         forgotTableView.register(explanationCell, forCellReuseIdentifier: "ExplanationTableViewCell")
         
-        let emailCell = UINib(nibName: "EmailTableViewCell", bundle: nil)
-        forgotTableView.register(emailCell, forCellReuseIdentifier: "EmailTableViewCell")
+        let textCell = UINib(nibName: "TextTableViewCell", bundle: nil)
+        forgotTableView.register(textCell, forCellReuseIdentifier: "reuseForText")
         
         let sendButtonCell = UINib(nibName: "SendButtonTableViewCell", bundle: nil)
         forgotTableView.register(sendButtonCell, forCellReuseIdentifier: "SendButtonTableViewCell")
```

Comment:
here is an extension below in 86-88 rows, that doesn't execute, probably better remove it

Reply:
It has resolved.